docs/server/commands.rst

```diff
@@ -500,6 +500,51 @@ Manually Installing Pootle
 These commands expose the database installation and upgrade process from the
 command line.
 
+.. django-admin:: init
+
+init
+^^^^
+
+Create the initial configuration for Pootle.
+
+Available options:
+
+:option:`--config`
+  The configuration file to write to.
+
+  Default: ``~/.pootle/pootle.conf``.
+
+.. versionchanged:: 2.7.1
```

Comment:
What about `versionadded` on each new option, or a versionadded block (http://sphinx-doc.org/markup/para.html#directive-versionadded) listing the new options?